README.md

```diff
@@ -662,7 +662,7 @@ It is expected that most users will be using the `Public` URL, however, it is al
 
 ## Release Notes
 
-[Changelist](./CHANGELIST.md)
```

Comment:
We should keep this, but it needs to point to [./RELEASES.md](the https://github.com/microsoft/ApplicationInsights-JS/blob/master/RELEASES.md) instead of ./CHANGELIST.md